apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -188,4 +230,16 @@ Page {
             searchInstalledNotice.show()
         }
     }
+
+    Component {
+        id: folderPickerPage
+
+        FolderPickerPage {
```

Comment:
Could this be set to start in the Downloads folder by default, or the folder last chosen by the user?

Reply:
I think it may work, but it requires changes to sailfish-pickers. Now if set default directory /home/user/MyFolder/, I will not be able to go to /home/user/

Reply:
New bug added to cover this; thanks @atatarov; I'm resolving this thread.